daemon/lib/swaps/client.go

```diff
@@ -0,0 +1,30 @@
+package swaps
+
+import (
+	"time"
+
+	"github.com/40acres/40swap/daemon/database/models"
+	"github.com/shopspring/decimal"
+)
+
+type SwapClient interface {
```

Comment:
Same, it can be  `Client` as you would use it as  `swaps.Client`

Reply:
Done 🏁